### ISSUE TITLE
Accessibility-WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (.indented:nth-child(4))

### ISSUE THREAD

**Issue by A11Ybot-OP** [SFTtech/openage#1444]
#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[openage - free, modern RTS game engine](https://openage.sft.mx/)
#### Element path
.indented:nth-child(4)
#### Snippet
`<p class="indented"><b>Modding:</b> openage uses an open API powered by our human-readable configuration language <a href="https://github.com/SFTtech/nyan/">nyan</a>.</p>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 1.37 (foreground color: #dcdcdc, background color: #ffffff, font size: 13.3pt (17.687px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Microsoft Edge version 96.0.1054.62
